baselayout/profile

```diff
@@ -22,10 +22,10 @@ umask 022
 
 # Set up PATH, all users get both bin and sbin to keep things simple.
 # Gentoo normally splits this up which is why the variable is called ROOTPATH
-export PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:${ROOTPATH}"
+export PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin${ROOTPATH:+:}${ROOTPATH-}"
```

Comment:
Dropping /sbin and /bin is fine in the image but are still needed in the SDK which never migrated to the all-in-/usr layout. So this is fine but when updating the ebuild please add a file to /etc/env.d that adds those dirs to ROOTPATH